pom.xml

```diff
@@ -844,11 +844,11 @@
             <version>2.2.2</version>
         </dependency>
         -->
+        <!-- https://mvnrepository.com/artifact/org.netbeans.external/jemmy-2.3.1.1 -->
```

Comment:
This POM works by pulling jemmy from the repository http://bits.netbeans.org/maven2/ (mvnrepository.com indexes other repositories). You probably don't need to note where on mvnrepository you downloaded it for ant.

Reply:
I actually copied the note from the maven central entry for this.